tools/c7n_index/c7n_index/metrics.py

```diff
@@ -128,35 +145,47 @@ def get_indexer(config, **kwargs):
 @indexers.register('es')
 class ElasticSearchIndexer(Indexer):
 
-    def __init__(self, config, **kwargs):
+    def __init__(self, config):
         self.config = config
-        self.es_type = kwargs.get('type', 'policy-metric')
-
-        host = [config['indexer'].get('host', 'localhost')]
+        # self.es_type = kwargs.get('type', 'policy-metric')
+        host = [config['indexer'].get('host')]
```

Comment:
Similar to default port number, is it OK to remove the default host `localhost`?

Reply:
since `host` and `port ` are required properties under `es` now, I assumed this wouldn't be needed, but since there are previous dependencies, I will add these back.